src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1974,19 +1055,19 @@ bool entitiesQuery
         }
       }
     }
-    else if (sco->type == SCOPE_TYPE_SIMPLE_QUERY)
+    else if (sco->type == SCOPE_TYPE_SIMPLE_QUERY)  // Used for string filters in V1
     {
-      // FIXME P4: Once issue #1705 is implemented, this check can be removed.
-      if (qStringFilters(sco->value, filters) != true)
+      stringFilterP = new StringFilter();
+      if (stringFilterP->parse(sco->value.c_str(), err) == false)
```

Comment:
If we add a StringFilter to the Scope class I understand that we will save doing parsing at mongoBackend level. Similar case to geo-things, that are parsed at getEntities() invoking the Scope fill() method:

```
if (scopeP->fill(ciP->apiVersion, geometry, coords, georel, &errorString) != 0)
```

(As far as I have checked, Scope.cpp/h haven't been modified in this PR... maybe I'm wrong, it seems that github has changed its web interface and now it is more difficult to see...)